src/libstd/sys/windows/ext/io.rs

```diff
@@ -10,6 +10,8 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
+//! Windows' IO module.
```

Comment:
What are these Windows IO comments doing in a rustdoc PR?

Reply:
It broke the build otherwise. There is a `#![deny(missing_docs)]` and rustdoc broke because of missing docs.

Reply:
That shouldn't have happened. All of `std::sys::windows` (and `std::os`, where it's re-exported) should be covered by `#[allow(missing_docs)]`:

https://github.com/rust-lang/rust/blob/39061ff67033399390930b9e7703a2e9e8312ced/src/libstd/sys/windows/mod.rs#L11

https://github.com/rust-lang/rust/blob/39061ff67033399390930b9e7703a2e9e8312ced/src/libstd/os/mod.rs#L14

If not having these docs broke the build, that means we just broke `#[allow(missing_docs)]`.

Reply:
Interesting. I'll look further then.

Reply:
Ok, the problem is a bit more complicated than that. Parser doesn't take into account windows mods and therefore doesn't apply its rules. However, rustdoc applies a cgf which it to access... `ext`! Therefore, `ext` doesn't have what the `windows.rs` has.